### ISSUE TITLE
Customizable NAT device subnet

### ISSUE THREAD

**Issue by evilhamsterman** [GNS3/gns3-gui#3615]
**Is your feature request related to a problem? Please describe.**
I'd like the ability to simulate multiple Internet sources. This can be useful for testing multihoming, multiple sites, etc.

**Describe the solution you'd like**
The ability to configure different subnets and DHCP services on NAT devices

**Describe alternatives you've considered**
Currently to replicate this if you just need to two it can be done using the Cloud and NAT devices, but sometimes you need more than two like multiple multi homed sites. In that case I've essentially created my own NAT devices using a router like VYOS but that's a lot of extra setup work.

**Additional context**

**Comment by grossmj** [comment]
Good idea. We use libvirt to create NAT devices on Linux (by connecting to `virbr0` interface by default).

As a workaround, you can edit the DHCP settings of that interface like this:

```
virsh net-destroy default
virsh net-edit default
```

Don't forget to restart it with `virsh net-start default`

**Comment by smart2128** [comment]
> **Is your feature request related to a problem? Please describe.** I'd like the ability to simulate multiple Internet sources. This can be useful for testing multihoming, multiple sites, etc.

I often use this scenario in my labs and I decided to permanently add a couple of new libvirt NAT networks for ISP1 (192.168.123.0/24) and ISP2 (192.168.124.0/24)
